backend/test/rules/simultaneous_test.clj

```diff
@@ -16,12 +16,12 @@
 (defexpect next-play
   ; Stores next-play
   (expect
-    {:player 0 :index 0 :row 0}
+    {:player 0 :index 0 :row 0 :target nil}
```

Comment:
I feel like this change should'nt be needed, nonexistence of the key should be treated as if it were nil.

Reply:
yet it is stored as `nil`. I'm not sure I understand what you'd expect here.

Reply:
🤔